client/pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>io.split.client</groupId>
         <artifactId>java-client-parent</artifactId>
-        <version>2.3.3-rc1</version>
+        <version>2.4.0</version>
```

Comment:
this should be a 3.0.0-rc1 actually. We are braking the API.

Reply:
done